cleanlab-studio-api/databricks.ipynb

```diff
@@ -37,7 +37,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install openai"
+    "%pip install openai\n",
+    "%pip install --upgrade cleanlab-studio"
```

Comment:
why are you including `--upgrade` in all of these?

We should assume these are for first-time users that don't have cleanlab studio installed, not our power-users.